src/clj/witan/send/charts.clj

```diff
@@ -30,57 +31,6 @@
 (defn load-transitions [path]
   (-> (tu/csv-to-dataset path sc/TransitionCounts) ds/row-maps))
 
-(def TH-setting->group
-  (hash-map :OOE "Other"
-            :MMSIB "Mainstream"
-            :MU "Other"
-            :MMSOB "Mainstream"
-            :MSSOB "Special"
-            :EO "Other"
-            :EYS "Mainstream"
-            :MMSIB "Mainstream"
-            :MMSOB "Mainstream"
-            :MAP "Other"
-            :MSS "Special"
-            :FEC "Further education"
-            :IMS "Mainstream"
-            :ISS "Special"
-            :ISSR "Special"
-            :MMSIB "Mainstream"
-            :MU "Other"
-            :MMSOB "Mainstream"
-            :MUOB "Other"
-            :MMS "Mainstream"
-            :MSSIB "Special"
-            :MSSOP "Special"
-            :MSSR "Special"
-            :MSSOB "Special"
-            :NMSS "Special"
-            :NMSSR "Special"
-            :MAP "Other"
-            :FEC "Further education"
-            :FEC "Further education"
-            :EO "Other"
-            :NON-SEND "Other"))
-
-(def Camden-setting->group
-  (hash-map :MSS "Special"
-            :IN "Mainstream"
-            :EO "Other"
-            :CC "Other"
-            :MU "Other"
-            :MMS "Mainstream"
-            :IMS "Mainstream"
-            :ISS "Special"
-            :ISSR "Special"
-            :PRU "Other"
-            :OOE "Other"
-            :FEC "Further education"
-            :ISC "Special"
-            :ISCR "Special"
-            :IT "Other"
-            :NON_SEND "Other"))
-
 (defn sankey [{:keys [title] :or {title ""}} df]
   (gg4clj/render
```

Comment:
interesting use of gg4clj - wasn't expecting it out of a gorilla form.
maybe we should extract the non-gorilla bits (ie running the r code and outputting to a file) into a util

Reply:
I thought about this, but at least at the moment this Sankey function is so specific to its use here. In the future we could maybe recreate it with more args to add more flexibility

Reply:
fair